src/tests/ftest/osa/offline_drain.py

```diff
@@ -48,7 +51,7 @@ def run_offline_drain_test(self, num_pool, data=False, oclass=None, pool_fillup=
             oclass = self.ior_cmd.dfs_oclass.value
 
         # Exclude target : random two targets  (target idx : 0-7)
-        exc = random.randint(0, 6)  # nosec
+        exc = random.randint(0, (self.hostlist_servers * self.engine_count))  # nosec
```

Comment:
Pre-existing, but we should replace `random` with `self.random`. Then we don't need `# nosec`
```suggestion
        exc = self.random.randint(0, (self.hostlist_servers * self.engine_count))
```